sonaranalyzer-dotnet/its/sources/ManuallyAddedNoncompliantIssues/NetFramework35/XmlReaderTest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Xml;
+
+namespace NetFramework35
+{
+    public class XmlReaderTest
+    {
+        public void XmlReader_EnableProhibitDtdAndSetResolver()
+        {
+            var settings = new XmlReaderSettings {ProhibitDtd = false, XmlResolver = new XmlUrlResolver()};
```

Comment:
you should also use in ITs 
```
    XmlReaderSettings settings = new XmlReaderSettings();
    settings.ProhibitDtd = false;
    settings.XmlResolver = new XmlSafeResolver();
```

Reply:
(initialization after the object creation)
(safe resolver as well)

Reply:
I've seen now that it's covered in the UTs

you can still add in one of the ITs

Reply:
Indeed it is covered in the unit tests but I've added now the case in the UTs